pkg/network/none/none.go

```diff
@@ -0,0 +1,60 @@
+package none
+
+import (
+	"context"
+	"os"
+	"os/exec"
+	"strconv"
+	"syscall"
+
+	"github.com/rootless-containers/rootlesskit/v2/pkg/api"
+	"github.com/rootless-containers/rootlesskit/v2/pkg/common"
+	"github.com/rootless-containers/rootlesskit/v2/pkg/messages"
+	"github.com/rootless-containers/rootlesskit/v2/pkg/network"
+)
+
+func NewParentDriver() (network.ParentDriver, error) {
+	return &parentDriver{}, nil
+}
+
+type parentDriver struct {
+}
+
+const DriverName = "none"
+
+func (d *parentDriver) MTU() int {
+	return 0
+}
+
+func (d *parentDriver) Info(ctx context.Context) (*api.NetworkDriverInfo, error) {
+	return &api.NetworkDriverInfo{
+		Driver: DriverName,
+	}, nil
+}
+
+func (d *parentDriver) ConfigureNetwork(childPID int, stateDir, detachedNetNSPath string) (*messages.ParentInitNetworkDriverCompleted, func() error, error) {
+	var cleanups []func() error
+
+	cmds := [][]string{
+		[]string{"nsenter", "-t", strconv.Itoa(childPID), "--no-fork", "-n", "-m", "-U", "--preserve-credentials", "ip", "address", "add", "127.0.0.1/8", "dev", "lo"},
+		[]string{"nsenter", "-t", strconv.Itoa(childPID), "--no-fork", "-n", "-m", "-U", "--preserve-credentials", "ip", "link", "set", "lo", "up"},
+	}
+	if err := common.Execs(os.Stderr, os.Environ(), cmds); err != nil {
+		return nil, nil, err
+	}
+
+	if detachedNetNSPath != "" {
```

Comment:
Got an error:
```console
$ rootlesskit --net=none --detach-netns bash
WARN[0000] [rootlesskit:parent] specifying --disable-host-loopback is highly recommended to prohibit connecting to 127.0.0.1:* on the host namespace (requires pasta, slirp4netns, or VPNKit) 
nsenter: reassociate to namespace 'ns/net' failed: Operation not permitted
[rootlesskit:parent] error: failed to setup network &{}: exit status 1
[rootlesskit:child ] error: EOF
```

Reply:
hmm, I never use `--detach-netns` actually, missed the test.

but then `none` network driver shouldn't have `--detach-netns` support since we don't have any network but `loopback` ?

Reply:
`none` may support `--detach-netns` for consistency with other network drivers, but not a hard requirement.

Reply:
if that so, then configuring `loppback` when `--detach-netns` seems useless.

Reply:
`rootlesskit --net=none --detach-netns bash` will run `bash` in the host netns, and detach a new lo-only netns in `$ROOTLESSKIT_STATE_DIR/netns`.

Reply:
got it, updated